contents/blog/posthog-vs-matomo.md

```diff
@@ -0,0 +1,435 @@
+- [Strengths of Matomo](#strengths-of-matomo)
+- [Coming soon to PostHog](#coming-soon-to-posthog)
+
+## How is PostHog different from Matomo?
+
+### 1. It's an all-in-one product analytics platform
+Matomo charges extra for product analytics features like funnel analysis, cohorts, path analysis, and session recording, and they're not tightly integrated. In PostHog, these are core features in all editions, including the open source release, while Experimentation is a no cost feature at [less than 1 million events per month](/pricing).
+
+### 2. Seamless integration with modern data stacks
+PostHog is built to work seamlessly with your data stack. That means we offer data import and export integrations with [most popular data warehouses](#integrations-and-data-sources), as well as numerous integrations with CRMs (Hubspot, Salesforce), data platforms (Segment, Airbyte, Rudderstack), engineering tools (PagerDuty, Sentry), and more via the [PostHog App Store](/apps).
+
+### 3. It's built for engineers
+PostHog is about giving engineering and product teams the tools they need to build better products. The core product analytics tools are part of this, but we go further by providing market-leading feature flag functionality, and integrating Session Recording so you can deploy one platform that does everything, rather than integrating multiple discrete tools into your stack.
+
+<ArrayCTA />
+
+## Feature comparison
+
+This table compares three self-hosted plans: Matomo On-Premise, PostHog Open Source, and PostHog Scale. 
+
+Matomo On-Premise ships with a robust set of core web analytics features, while advanced product analytics features are available at various prices from the Matomo On-Premise Marketplace.
+
+All features in PostHog Open Source are totally free, including core product analytics tools. 
+
+PostHog Scale is a paid edition that adds Experimentation, Group Analytics, support for multiple projects, and numerous team management tools. It's priced on a per event basis and is free up to 1 million events per month.
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+	<thead>
+    	<tr>
+          <td className="w-3/12"></td>
+        	<td className="w-3/12 text-center"><strong>Matomo On-Premise</strong></td>
+        	<td className="w-3/12 text-center"><strong>PostHog Open Source</strong></td>
+        	<td className="w-3/12 text-center"><strong>PostHog Scale</strong></td>
+    	</tr>
+	</thead>
+	<tbody>
+    	<tr>
+        	<td><strong>Platform</strong></td>
+        	<td></td>
+        	<td></td>
+        	<td></td>
+    	</tr>
+		<tr>
+			<td>Open source</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+    	<tr>
+        	<td>Host yourself</td>
```

Comment:
Any reason we call this 'Host yourself' instead of 'Self-host' as we do everywhere else?

Reply:
Honestly, I just prefer the phrase in this context. We do use on the homepage, so it's not like we never use it. I dunno, it just reads nicer? Less formal. It's entirely subjective choice!